src/pages/DogNew.test.js

```diff
@@ -17,20 +17,9 @@ describe("When DogNew Renders", () => {
     const heading = dogNew.find('h1')
     expect(heading.text()).toEqual('Create A New Dog')
   })
-  it("displays a form", () => {
-    const form = dogNew.find('Form')
+  it("displays a dog form", () => {
+    const form = dogNew.find('DogForm')
     expect(form.length).toEqual(1)
   })
-  it("displays an input for age", () => {
-    const ageInput = dogNew.find('[name="age"]')
-    expect(ageInput.length).toEqual(1)
-  })
-  it("displays an input for enjoys", () => {
-    const enjoysInput = dogNew.find('[name="enjoys"]')
-    expect(enjoysInput.length).toEqual(1)
-  })
-  it("displays an input for image", () => {
-    const imageInput = dogNew.find('[name="image"]')
-    expect(imageInput.length).toEqual(1)
-  })
+
```

Comment:
You will benefit from having more tests than this. I'll approve but think about spending some time coming up with input tests.